lib/config/options/index.ts

```diff
@@ -245,10 +245,14 @@ const options: RenovateOptions[] = [
   {
     name: 'dryRun',
     description:
-      'If enabled, perform a dry run by logging messages instead of creating/updating/deleting branches and PRs.',
-    type: 'boolean',
+      '"extract": Performs extract only, prints results, exits.\n' +
+      '"lookup": Performs extract and lookup, but exits before running through branches.\n' +
+      '"full": Perform a dry run by logging messages instead of creating/updating/deleting branches and PRs.\n' +
+      'Default value = null',
```

Comment:
This level of detail should be moved to the markdown doc

Reply:
do you mean it should appear under the table?

Reply:
something like this?
![image](https://user-images.githubusercontent.com/98313426/157402138-918ba401-c11c-41af-8b25-224c37a1c06f.png)